Yoast/javascript
# LINGO-531 Add sentence beginning assessment for Hungarian

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds the sentence beginning assessment and specs for Hungarian.
* [Yoast SEO Free] Implements the consecutive sentences assessment for Hungarian to ensure variety in a text.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set your website to Hungarian 
* Test in Yoast SEO Free
* Create a new post and add three or more Hungarian sentences (you can use [this fake texts generator](https://www.lipsum.com/)):

     * that all begin with the same written-out number, like `egy` ("one")
       E.g. `Egy macska. egy kutya. egy teknős.`

     * that all begin with the same demonstrative pronoun, like `ez` ("this").
       E.g. `Ez a ház csinos. ez az ajtó nagy. ez az erkély nagyszerű kilátást nyújt.`

* In each of the above cases, the sentence beginning readability assessment should read: `Consecutive sentences: There is enough variety in your sentences. That's great!`

* Write three or more Hungarian sentences that begin with the same word.
   E.g. `Reggel van. Reggel álmos vagyok. Reggel dolgozni kell.`

* The sentence beginning readability assessment should read: `Consecutive sentences: The text contains 3 consecutive 
  sentences starting with the same word. Try to mix things up!`

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
